Jupyter Notebook/src/CartoCosmos.py

```diff
@@ -0,0 +1,283 @@
+from ipyleaflet import *
+import ipywidgets as widgets
+import math as Math
+import json
+import geojson
+import shapely.geometry as geo
+import shapely.wkt
+
+class Planetary_Maps:
+    def __init__(self, targetName):
+        self.targetName = targetName
+        self.json_file = 'geoServerLayers.json'
+        self.base_layers = []
+        self.planet_map = None
+        self.map_layers = {
+            'base': [],
+            'overlays': []
+            }
+        self.longitudeRange = None
+        self.latDomain = None
+        self.longitudeDirection = None
+        self.latLonLabel = None
+        self.drawLabel = None
+        self.wktTextBox = None
+        self.wktButton = None
+        self.dMajorRadius = 0
+        self.dMinorRadius = 0
+        self.Find_Radius()
+        self.Create_Layers()
+        self.Create_Widgets()
+        self.Create_Map()
+        self.feature_collection = {
+            'type': 'FeatureCollection',
+            'features': []
+        }
+
+    def Find_Radius(self):
```

Comment:
Might actually be a better idea to parse the JSON USGS gave us to grab the radii from. This way, we don't have to store radii for every target